internal/policies/mount/adsys_mount/Cargo.toml

```
@@ -0,0 +1,21 @@
+[package]
+name = "adsys_mount"
+description = "Binary for handling the user mounts policy of adsys.\nThis is not meant to be executed manually."
+version = "0.1.0"
+edition = "2021"
+include = ["src/**/*"]
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+[lib]
+name = "lib"
+path = "src/lib.rs"
+
+[dependencies]
+anyhow = "1.0.65"
+chrono = "0.4.22"
+gio = "0.16.2"
+glib = "0.16.2"
+log = "0.4.17"
+serde = { version = "1.0.147", features = ["derive"] }
+serde_json = "1.0.87"
+thiserror = "1.0.37"
```

I recommend referencing their major version for crates with stabilized versions (according to the semantic versioning). Like this:

```suggestion
serde = { version = "^1", features = ["derive"] }
serde_json = "^1"
thiserror = "^1"
```